bundles/org.eclipse.swt/Eclipse SWT Drag and Drop/common/org/eclipse/swt/dnd/DNDListener.java

```diff
@@ -13,17 +13,18 @@
  *******************************************************************************/
 package org.eclipse.swt.dnd;
 
-import org.eclipse.swt.internal.*;
+import java.util.*;
+
 import org.eclipse.swt.widgets.*;
 
 
 class DNDListener extends TypedListener {
 	Widget dndWidget;
 /**
  * DNDListener constructor comment.
- * @param listener org.eclipse.swt.internal.SWTEventListener
+ * @param listener java.util.SWTEventListener
```

Comment:
```suggestion
 * @param listener java.util.EventListener
```

Or even better just remove the type from the java-doc or even the entire `@param` line as it has no value.

Reply:
param removed.